csharp/src/Google.OpenLocationCode/Constants.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Google.OpenLocationCode
+{
+    internal static class Constants
+    {
+        public const char Separator = '+';
+        public const int SeparatorIndex = 0x08;
+        public const char PaddingSuffix = '0';
+
+        public static readonly char[] Alphabet = { '2', '3', '4', '5', '6', '7', '8', '9', 'C', 'F', 'G', 'H', 'J', 'M', 'P', 'Q', 'R', 'V', 'W', 'X' };
+        public static readonly ReadOnlyDictionary<char, int> AlphabetIndex = new ReadOnlyDictionary<char, int>( new Dictionary<char, int>
```

Comment:
This data structure seems to duplicate the information that is contained in Alphabet. Other implementations look up the position of a digit in the Alphabet array and use that as the value. Is that possible?